app/src/App/tests/AppBackendActive.test.tsx

```diff
@@ -9,6 +9,16 @@ import App from '../App';
 
 jest.mock('../../configs/env', () => ({
   BACKEND_ACTIVE: true,
+  OPENSRP_ROLES: {
+    USERS: 'ROLE_EDIT_KEYCLOAK_USERS',
+    PLANS: 'ROLE_VIEW_KEYCLOAK_USERS',
+    LOCATIONS: 'ROLE_VIEW_KEYCLOAK_USERS',
+    CARD_SUPPORT: 'ROLE_VIEW_KEYCLOAK_USERS',
+    INVENTORY: 'ROLE_VIEW_KEYCLOAK_USERS',
+    TEAMS: 'ROLE_VIEW_KEYCLOAK_USERS',
+    PRODUCT_CATALOGUE: 'ROLE_VIEW_KEYCLOAK_USERS',
+    FORM_CONFIGURATION: 'ROLE_VIEW_KEYCLOAK_USERS',
```

Comment:
Does any of these allow an array.

Reply:
it will be list of comma separated roles, then in code we will do a `.split(',')` to make it an array @dubdabasoduba